Model/Sales/Order/Creditmemo/Total/ShippingProtection.php

```diff
@@ -6,17 +6,22 @@
 
 namespace Extend\Integration\Model\Sales\Order\Creditmemo\Total;
 
+use Extend\Integration\Api\Data\ShippingProtectionInterface;
 use Extend\Integration\Api\ShippingProtectionTotalRepositoryInterface;
 use Magento\Sales\Model\Order\Creditmemo;
+use Extend\Integration\Model\ShippingProtectionFactory;
 
 class ShippingProtection extends \Magento\Sales\Model\Order\Creditmemo\Total\AbstractTotal
 {
     private ShippingProtectionTotalRepositoryInterface $shippingProtectionTotalRepository;
+    private ShippingProtectionFactory $shippingProtectionFactory;
 
     public function __construct(
-        ShippingProtectionTotalRepositoryInterface $shippingProtectionTotalRepository
+        ShippingProtectionTotalRepositoryInterface $shippingProtectionTotalRepository,
+        ShippingProtectionFactory $shippingProtectionFactory
     ){
         $this->shippingProtectionTotalRepository = $shippingProtectionTotalRepository;
+        $this->shippingProtectionFactory = $shippingProtectionFactory;
```

Comment:
What is the factory doing here? Just needed for the base class?

Reply:
It's used on line 89